algorithms/src/merkle_tree/merkle_path.rs

```diff
@@ -31,10 +31,8 @@ pub type MerkleTreeDigest<P> = <<P as MerkleParameters>::H as CRH>::Output;
 
 /// Stores the hashes of a particular path (in order) from leaf to root.
 /// Our path `is_left_child()` if the boolean in `path` is true.
-#[derive(Derivative)]
```

Comment:
Is there a reason to change this? In particular now we're comparing the `Parameters`, I think.

Reply:
The check on `Parameters` will check the instance of the CRH (which itself is empty) and the string.

Reply:
Ray is planning to tackle the Merkle tree with a 2-to-1 refactor this week